### PR TITLE
app creation with build process monitor

### PR DESCRIPTION
**Pull request by BlueShells**
need to talk with someone with this

### CODE REVIEW

File: scripts/monitoring/cron-send-build-app.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    OpenShiftOC.new_project(proj_name, verbose)
+
+    OpenShiftOC.new_app(app, proj_name, verbose)
+    #1 is error
+    create_app = 1
+    # Now we wait until the pod comes up
+    for _ in range(24):
+        time.sleep(5)
+        #checking the building pod 
+        buildPod=OpenShiftOC.get_build_pod('nodejs-example',proj_name,verbose)
```
Comment (kwoodson) [inline scripts/monitoring/cron-send-build-app.py]: This should be:
----
+        buildPod=OpenShiftOC.get_build_pod(app,proj_name,verbose)
----

File: scripts/monitoring/cron-send-build-app.py [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+        if buildPod and buildPod['status']:
+            if verbose:
+               print buildPod['status']['phase']
+
+        #checking the status of buildpod if it is error
+        if buildPod and buildPod['status']['phase'] == 'Error':
+            print 'fail'
+            break
+
+        pod = OpenShiftOC.get_pod('nodejs-example', proj_name, verbose)
```
Comment (kwoodson) [inline scripts/monitoring/cron-send-build-app.py]: pod = OpenShiftOC.get_pod(app, proj_name, verbose)

**Comment by BlueShells** [comment]
@kwoodson ,I need some the next step ,we now still package ourself manual or you guys will package at a certain time?

File: scripts/monitoring/cron-send-build-app.py [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+            if httpstatus == 200:
+                create_app_func = 0
+    return create_app_func
+
+def main():
+    ''' Do the application creation
+    '''
+    proj_name = 'ops-monitor-appbuild' + os.environ['ZAGG_CLIENT_HOSTNAME']
+    app = 'nodejs-example'
+    verbose = False
```
Comment (kwoodson) [inline scripts/monitoring/cron-send-build-app.py]: Delete this line

File: scripts/monitoring/cron-send-build-app.py [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+                if create_app == 0:
+                    CreateTime = time.time() - start_time
+                    print 'success'
+                    print 'Time: %s' % CreateTime
+                    print 'BuildTime: %s' % BuildTime
+                    break
+            if create_app == 0:
+                break
+    else:
+        if verbose:
```
Comment (kwoodson) [inline scripts/monitoring/cron-send-build-app.py]: Remove if verbose

File: scripts/monitoring/cron-send-build-app.py [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+            print 'Time: %s' % str(time.time() - start_time)
```
Comment (kwoodson) [inline scripts/monitoring/cron-send-build-app.py]: indent <<

File: scripts/monitoring/cron-send-build-app.py [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+            print 'fail'
```
Comment (kwoodson) [inline scripts/monitoring/cron-send-build-app.py]: indent <<

File: scripts/monitoring/cron-send-build-app.py [hunk 7]
```diff
@@ -0,0 +1,220 @@
...
+                if create_app == 0:
+                    CreateTime = time.time() - start_time
+                    print 'success'
+                    print 'Time: %s' % CreateTime
+                    print 'BuildTime: %s' % BuildTime
+                    break
+            if create_app == 0:
+                break
+    else:
+        print 'Time: %s' % str(time.time() - start_time)
```
Comment (kwoodson) [inline scripts/monitoring/cron-send-build-app.py]: Capture the time here and return it for the build time even on failure

File: scripts/monitoring/cron-send-build-app.py [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+                    CreateTime = time.time() - start_time
+                    print 'success'
+                    print 'Time: %s' % CreateTime
+                    print 'BuildTime: %s' % BuildTime
+                    break
+            if create_app == 0:
+                break
+    else:
+        BuildTime = time.time() - start_time
+        print 'Time: %s' % str(time.time() - start_time)
```
Comment (kwoodson) [inline scripts/monitoring/cron-send-build-app.py]: Can remove this line.

File: scripts/monitoring/cron-send-build-app.py [hunk 9]
```diff
@@ -0,0 +1,223 @@
...
+    create_app = 1
+    BuildTime = 0
+    CreateTime = 0
+    # Now we wait until the pod comes up
+    for _ in range(24):
+        time.sleep(10)
+        #checking the building pod
+        buildPod = OpenShiftOC.get_build_pod(app, proj_name, verbose)
+        if buildPod and buildPod['status']['phase'] == 'Failed':
+            CreateTime = time.time() - start_time
```
Comment (kwoodson) [inline scripts/monitoring/cron-send-build-app.py]: Should this be buildtime?

**Comment by kwoodson** [comment]
:+1:

**Pull request merged**